docs/specs/ecma2Yaml.yml

```diff
@@ -18,7 +18,8 @@ repos:
             },
           }],
         }
-      docfx.yml:
+      docfx.yml: |
+          sourceMap: .sourceMap.json
```

Comment:
`OpsConfigLoader` should auto populate this property, otherwise this needs config change in repos.

Reply:
https://github.com/dotnet/docfx/pull/5687#discussion_r399924939
I thought a config change is needed...

Reply:
Oh, I mean we'll expose a config in docfx as a feature, but for existing OPS repos, it'll have a thin compact layer in OpsConfigLoader. 

We aren't touching any of the user configs during the migration.

Reply:
OK, got your point